src/engine/runtime.js

```diff
@@ -1737,7 +1745,6 @@ class Runtime extends EventEmitter {
         // Script glows must be cleared.
         this._scriptGlowsPreviousFrame = [];
         this._updateGlows();
-        this.requestTargetsUpdate(editingTarget);
```

Comment:
This is ok to do because everywhere that calls `runtime.setEditingTarget` already also calls `emitTargetsUpdate` itself.